src/components/Global/PageTitle.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <header class="page-title">
+    <h1>{{ pageTitle }}</h1>
+    <p v-if="pageDescription">{{ pageDescription }}</p>
+  </header>
+</template>
+
+<script>
+export default {
+  name: 'PageTitle',
+  props: ['pageTitle', 'pageDescription']
```

Comment:
Also wondering if we need to be this verbose in our prop naming? Is there a reason we wouldn't use the prop name `title` opposed to `propTitle`? Same with description. This is not a big deal, just curious

Reply:
yeah, agree, will plan to remove title prop all together and change to description